source/agent/analytics/analytics-agent.js

```diff
@@ -114,7 +114,8 @@ class AnalyticsAgent extends BaseAgent {
       // check if this algorithm has output
       if (this.algorithms[algo].outputfourcc === 'I420') {
         // generated a new stream
-        const newStreamId = Math.random() * 1000000000000000000 + '';
+        //const newStreamId = Math.random() * 1000000000000000000 + '';
+        const newStreamId = 'A0A0'+options.media.video.from;
```

Comment:
Previously a stream can be analyzed twice (even with same parameters, algorithm, etc), then generated two new stream with different ID.
This makes all generated stream have the same ID.
I suggest construct generated stream ID using more information, and get those information from conference agent through RPC if needed.